server/lib/src/plugins/refint.rs

```diff
@@ -491,28 +492,47 @@ mod tests {
     // The create references a uuid that does exist - validate
     #[test]
     fn test_create_uuid_reference_exist() {
-        let ea: Entry<EntryInit, EntryNew> = Entry::unsafe_from_entry_str(
-            r#"{
-            "attrs": {
-                "class": ["group"],
-                "name": ["testgroup_a"],
-                "description": ["testgroup"],
-                "uuid": ["d2b496bd-8493-47b7-8142-f568b5cf47ee"]
-            }
-        }"#,
+        let ea = entry_init!(
+            (Attribute::Class, EntryClass::Group.to_value()),
+            (Attribute::Name, Value::new_iname("testgroup_a")),
+            (Attribute::Description, Value::new_utf8s("testgroup")),
+            (
+                Attribute::Uuid,
+                Value::Uuid(Uuid::parse_str(TEST_TESTGROUP_A_UUID).unwrap())
+            )
         );
-
-        let eb: Entry<EntryInit, EntryNew> = Entry::unsafe_from_entry_str(
-            r#"{
-            "attrs": {
-                "class": ["group"],
-                "name": ["testgroup_b"],
-                "description": ["testgroup"],
-                "member": ["d2b496bd-8493-47b7-8142-f568b5cf47ee"]
-            }
-        }"#,
+        let eb = entry_init!(
+            (Attribute::Class, EntryClass::Group.to_value()),
+            (Attribute::Name, Value::new_iname("testgroup_b")),
+            (Attribute::Description, Value::new_utf8s("testgroup")),
+            (
+                Attribute::Member,
+                Value::Refer(Uuid::parse_str(TEST_TESTGROUP_A_UUID).unwrap())
+            )
         );
 
+        // let ea: Entry<EntryInit, EntryNew> = Entry::unsafe_from_entry_str(
```

Comment:
Just delete these?

Reply:
oh yeah, testing artifact. will be gone.